plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/meta/CubridMetaModelDescriptor.java

```diff
@@ -0,0 +1,112 @@
+package org.jkiss.dbeaver.ext.cubrid.model.meta;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.jkiss.code.NotNull;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.cubrid.model.CubridSQLDialect;
+import org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaObject;
+import org.jkiss.dbeaver.model.impl.AbstractDescriptor;
+import org.jkiss.dbeaver.model.sql.SQLDialectMetadata;
+import org.jkiss.dbeaver.model.sql.registry.SQLDialectRegistry;
+import org.jkiss.utils.ArrayUtils;
+import org.jkiss.utils.CommonUtils;
+
+public class CubridMetaModelDescriptor extends AbstractDescriptor {

```

Comment:
I do not see any reason for the existence of this class. Our GenericMetaModelDescriptor is working well. And you have all the necessary ID-s in your plugin.xml. So, please remove it.